### ISSUE TITLE
3rd bin selection did not perform, because there is only one assembly!

### ISSUE THREAD

**Issue by Enewmoon** [EMBL-PKU/BASALT#37]
Dear @EMBL-PKU ，
I have multiple sites, but only one sample per site.
So my code is BASALT -a Y_assembly.fa -s Y.1.fastq,Y.fastq -t 64 -m 250

However, when I run to the third step, the following problem appears, can I skip the third step or how to adjust the code?

Basalt_checkpoint.txt 
1st autobinner done!
2nd bin selection within group done!
3rd bin selection did not perform, because there is only one assembly!

Looking forward to your early reply.

**Comment by EMBL-PKU** [comment]
Hi,
You do not need to adjust the code for that. The selection module for multiple assemblies could be activated only when there were several assemblies presented. If there is only one assembly, as you already found it, BASALT would skip the step. So do not worry, it is ok

**Comment by Enewmoon** [comment]
Thank you very much for your answer.

**Comment by Enewmoon** [comment]
Hello, bother you again, it doesn't seem to run after the third step, but there should be many more steps after that, how to solve this.
Looking forward to your early reply.
![b3b66549a0a77ba935be00a8491dc13](https://github.com/EMBL-PKU/BASALT/assets/140048330/9be31c76-8305-48d3-80cd-bbb2f4b982a9)

**Comment by EMBL-PKU** [comment]
Is there only one paired-end reads present in your command? This error may occasionally occurred if only one PE-reads present. I am trying to solve this issue. It may take one day. Please wait until 0607-0608 2024, I will update the new script once I finished.

**Comment by Enewmoon** [comment]
> Is there only one paired-end reads present in your command? This error may occasionally occurred if only one PE-reads present. I am trying to solve this issue. It may take one day. Please wait until 0607-0608 2024, I will update the new script once I finished.

Yes, I only have one paired-end reads present. Thank you again for your reply.
